static/js/src/renewal-modal.js

```diff
@@ -368,7 +368,7 @@ function handleIncompleteRenewal(renewal) {
   let subscriptionStatus;
 
   if (renewal.stripeInvoices) {
-    invoice = renewal.stripeInvoices[renewal.stripeInvoices.length - 1];
+    invoice = renewal.stripeInvoices[0];
```

Comment:
If `renewal.stripeInvoices` is an empty array, `invoice` would end up as undefined, causing an error.

I think if it were an empty array, ua-contracts would simply not return it and it would be undefined. But it's better to be resilient and instead check for what you are really expecting, an array with one element:
```js
if (renewal.stripeInvoices && renewal.stripeInvoices.length > 1) {
    ...
}
```

Reply:
Good idea - updated